db/migrate/20200615134519_add_title_and_description_to_saved_scenarios.rb

```diff
@@ -0,0 +1,45 @@
+class AddTitleAndDescriptionToSavedScenarios < ActiveRecord::Migration[5.2]
+  def up
+    add_column :saved_scenarios, :title, :string
+    add_column :saved_scenarios, :description, :text
```

Comment:
I don't think the title should be optional. I would expect the user to provide a title for the scenario; without one I think it won't be clickable in the scenario list table?

If you agree, can you also add a presence validation for `title` on SavedScenario and validation error-handling on the form?

It would also be nice to insert these columns after `scenario_id_history` column, rather than the default at the end of the table:

```suggestion
    add_column :saved_scenarios, :title, :string, null: false, after: :scenario_id_history
    add_column :saved_scenarios, :description, :text, after: :title
```

Reply:
The title validation was already there on SavedScenario, but I'll also add a `required` tag to the form - good one! I'll also add the changes you suggested above for the positioning